addons/sys_prc117f/fnc_preset_information.sqf

```diff
@@ -88,7 +88,7 @@ for "_i" from 0 to 99 do {
     HASH_SET(_channel,"power",20000);
     HASH_SET(_channel,"encryption",0);
     HASH_SET(_channel,"channelMode", "BASIC");
-    private _desc = format["FMLOSVOC%1",([_i+1, 2] call CBA_fnc_formatNumber)];
+    private _desc = format["NET%1",([_i+1, 2] call CBA_fnc_formatNumber)];
```

Comment:
Not sure we want to change the default channel names at this time.

Reply:
Revert issue, had code there earlier. Nevertheless, FMLOSVOC is coming from the PRC-152, the "default" preset uses the correct name. So the other sides receive incorrect names meant for the PRC-152 where the format of "NETXX" is supposed to be used. Thoughts?

Reply:
Yeah, you're right. These should indeed before NETXX for the 117F, I was a little puzzled as it wasn't reference in the PR description.

This is definitely worth putting in the "When merged this pull request will:" section of the PR as a bullpoint as it should probably go in the changelog as fix as it be some legacy missions briefings reference these values.

Reply:
I've added it to the PR. My first attempt at fixing the name bug was to patch the name in the preset_information. Off course that doesn't work when someone doesn't use those defaults at all. After reverting those changes that was fixed, hence not in the original PR.